src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/MouseEventSender.cs

```diff
@@ -366,7 +366,63 @@ public static void SendMouseEvent(int replaySegment, MouseInputActionData mouseI
+        /// <param name="scroll">Simulates mouse scroll wheel movement.</param>
+        public static void SendMouseEvent(Transform mouseTarget,
+                                          bool leftButton=false,
+                                          bool middleButton=false,
+                                          bool rightButton=false,
+                                          bool forwardButton=false,
+                                          bool backButton=false,
+                                          Vector2 scroll = default)
+        {
+            Vector2 screenPosition = Camera.main.WorldToScreenPoint(mouseTarget.position);
+            #if ENABLE_LEGACY_INPUT_MANAGER
+            SendMouseEventLegacy(screenPosition, Vector2.zero, scroll, leftButton, middleButton, rightButton, forwardButton, backButton);
+            #else
+            var screenWidth = Screen.width;
+            var screenHeight = Screen.height;
+            var normalizedPosition = new Vector2(screenPosition.x / screenWidth, screenPosition.y / screenHeight);
+            SendRawPositionMouseEvent(0, normalizedPosition, leftButton, middleButton, rightButton, forwardButton, backButton, scroll);
+            #endif
+        }
+
+        /// <summary>
+        /// Simulates a mouse event at the specified screen position. Bottom left is (0,0) and top right is (Screen.width, Screen.height).
+        /// </summary>
+        /// <param name="mouseScreenPosition">The screen coordinates where the mouse event will be simulated.</param>
+        /// <param name="leftButton">Simulates left mouse button press if true.</param>
+        /// <param name="middleButton">Simulates middle mouse button press if true.</param>
+        /// <param name="rightButton">Simulates right mouse button press if true.</param>
+        /// <param name="forwardButton">Simulates forward mouse button press if true.</param>
+        /// <param name="backButton">Simulates back mouse button press if true.</param>
+        /// <param name="scroll">Simulates mouse scroll wheel movement.</param>
+        public static void SendMouseEvent(Vector2 mouseScreenPosition,
+                                          bool leftButton=false,
+                                          bool middleButton=false,
+                                          bool rightButton=false,
+                                          bool forwardButton=false,
+                                          bool backButton=false,
+                                          Vector2 scroll = default)
+        {
+            #if ENABLE_LEGACY_INPUT_MANAGER
+            SendMouseEventLegacy(mouseScreenPosition, Vector2.zero, scroll, leftButton, middleButton, rightButton, forwardButton, backButton);
+            #else
+            var screenWidth = Screen.width;
+            var screenHeight = Screen.height;
+            var normalizedPosition = new Vector2(mouseScreenPosition.x / screenWidth, mouseScreenPosition.y / screenHeight);
```

Comment:
I don't understand this logic.  It looks like it is computing a fractional percentage of the screen dimensions for the mouse instead of just sending the coordinates ?

Send Raw Position mouse event already expects the mouse position in terms of the width/height.  This code doesn't seem like it creates the correct mouse inputs

Reply:
Fixed. It was wrong. When I was reading the code I understood it correctly. It would be helpful if we at least add input/return type docs especially for things like this.